packages/dao/flattened_contracts/FIFSResolvingRegistrar.sol

```diff
@@ -0,0 +1,124 @@
+// File: @aragon/os/contracts/lib/ens/AbstractENS.sol
```

Comment:
Btw, why do we need to add a dir with flattened contracts?

Reply:
Some Aragon dependencies use it